test/services/events/EventsConfigRepositorySpec.scala

```diff
@@ -0,0 +1,124 @@
+/*
+ * Copyright 2017 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package services.events
+
+import model.persisted.eventschedules._
+import org.scalatest.concurrent.ScalaFutures
+import repositories.events.EventsConfigRepository
+import services.BaseServiceSpec
+import org.joda.time.{ LocalDate, LocalTime }
+import org.scalatest.Matchers
+import org.scalatest.time.{ Millis, Seconds, Span }
+import org.scalatestplus.play.PlaySpec
+import uk.gov.hmrc.play.test.UnitSpec
+
+class EventsConfigRepositorySpec extends UnitSpec with Matchers with ScalaFutures {
+  "events" must {
+    "successfully parse the config" in {
+      val input = """- eventType: FSAC
```

Comment:
I believe it would be useful to have test that load yams file (if possible) and parse it (at least to make sure that file we submitted is correct).